addons/payment/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,54 @@
+---Models in module 'payment'---
+new model payment.link.wizard [transient]
+# NOTHING TO DO
+
+---Fields in module 'payment'---
+payment      / account.invoice          / transaction_ids (many2many)   : DEL relation: payment.transaction
+payment      / account.move             / transaction_ids (many2many)   : NEW relation: payment.transaction
+payment      / payment.transaction      / invoice_ids (many2many)       : relation is now 'account.move' ('account.invoice') [nothing to do]
+# DONE: post-migration updated new ids
+
+payment      / payment.acquirer         / auth_msg (html)               : NEW hasdefault
+payment      / payment.acquirer         / check_validity (boolean)      : NEW
+payment      / payment.acquirer         / color (integer)               : NEW isfunction: function, stored
+payment      / payment.acquirer         / display_as (char)             : NEW
+# NOTHING TO DO
+
+payment      / payment.acquirer         / environment (selection)       : DEL required, selection_keys: ['prod', 'test'], req_default: function
+payment      / payment.acquirer         / state (selection)             : NEW required, selection_keys: ['disabled', 'enabled', 'test'], req_default: function, hasdefault
+# DONE: Pre-migration renamed to state
```

Comment:
1. move `payment      / payment.acquirer         / website_published (boolean)   : DEL` here
2. add comment like this:
Post-migration: 
  set `enabled` where environment == prod and website_published True
  set `disabled` where environment == prod and website_published False